[OSDOCS-9157]:What's New Announcement for new force-delete cluster flag "--best-effort" (#69657)
openshift/openshift-docs: rosa_release_notes/rosa-release-notes.adoc

=== Q4 2023

* **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to a new version. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.32[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].

> * **Delete cluster command enhancement.** With the release of ROSA CLI (`rosa`) version 1.2.31, a new argument, `--best-effort`, was added that allows you to force-delete clusters when using the `rosa delete cluster` command. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-delete-cluster_rosa-managing-objects-cli[delete cluster].

Reviewer: ```suggestion
* **Delete cluster command enhancement.** With the release of ROSA CLI (`rosa`) version 1.2.31, the `--best-effort` argument was added, which allows you to force-delete clusters when using the `rosa delete cluster` command. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-delete-cluster_rosa-managing-objects-cli[delete cluster].
```
